Source/DafnyLanguageServer.Test/Lookup/HoverVerificationTest.cs

```diff
@@ -194,7 +195,10 @@ method Abs(x: int) returns (y: int)
 
     [TestMethod, Timeout(5 * MaxTestExecutionTimeMs)]
     public async Task IndicateSlowHints() {
-      var documentItem = await GetDocumentItem(@"
+      var original = DafnyHoverHandler.RuLimitToBeOverCostly;
+      DafnyHoverHandler.RuLimitToBeOverCostly = 500000;
```

Comment:
Is this bounding the resource count (given the `Ru` prefix)? If so I'm confused as to why the test has been flaky.

Reply:
The problem came that, when I designed the test originally, the verification of the slow to verify method was taking 15M Ru, and I thought this number was stable. However, after running it a few more times, and including looking at the error on the CI build, it looks like verification goes down to using only 7M resources sometimes, and the hard-coded limit is 10M. Hence, I manually set the limit to 500K so that it's always too slow for this test.

Reply:
For anyone else curious/anxious about this, we realized the resource count was only varying because of the `{:timeLimit ...}` attribute. Naturally if verification is timing out because of a wall-clock time limit, the resource count will vary. :)